.github/workflows/main.yml

```diff
@@ -0,0 +1,25 @@
+name: deploy:heroku:create-app
+
+on:
+  push:
+    branches:
+      - 'release/**'
+
+jobs:
+  create_app:
+    runs-on: ubuntu-latest
+    steps:
+      - name: git-checkout
+        uses: actions/checkout@v2
+
+      - name: heroku-create-app
```

Comment:
We discussed that we will not create app, rather we will only do deployment to avoid issues around managing env variables whenever app is created.

Can we ensure we are not creating new app, but only do a deployment?

Reply:
@jjalan  sir, If the app does not exist then it will create a new app for first time, but after that If again a new release branch is created it will update the app to deploy that new branch.

Reply:
Understood.

Let's remove the app creation and only do deploy. I would like that step to be explicit so it would dev a chance to set env variables explicitly.

We should rename it to deploy-app

Reply:
done